src/inmanta/server/config.py

```diff
@@ -183,6 +182,14 @@ def validate_fact_renew(value):
     is_list,
 )
 
+server_access_control_allow_origin = Option(
+    "server",
+    "access-control-allow-origin",
+    None,
+    "Configures the access_control_allow_origin setting of the http server."
+    "Defaults to not sending an AOCA header.",
+    is_str_opt
```

Comment:
Is this enough or do we need to check on valid values? At least newlines and other strange characters should not be allowed because it could screw up requests seriously. (Security is less of an issue, because if you control the config you control the entire server anyway).